appveyor.yml

```diff
@@ -57,6 +57,10 @@ environment:
   # "(converted from warning) unable to identify current timezone 'C':" for an unknown reason.
   # This environment variable works around to test SparkR against a higher version.
   R_REMOTES_NO_ERRORS_FROM_WARNINGS: true
+  # AppVeyor does not have python3 yet which is used by default.
+  PYSPARK_PYTHON: python
+  # TODO(SPARK-32453): Remove SPARK_SCALA_VERSION environment and let load-spark-env scripts detect it.
+  SPARK_SCALA_VERSION: 2.12
```

Comment:
Just a question. Is this enough to fix AppVeyor build?

Reply:
I'm testing now.. :). If it needs a more complicated fix, I'll just revert this bit and merge.